src/core/lib/iomgr/event_engine/endpoint.cc

```diff
@@ -116,8 +116,10 @@ absl::string_view endpoint_get_peer(grpc_endpoint* ep) {
     return "";
   }
   if (eeep->peer_address.empty()) {
+    auto uri = ResolvedAddressToURI(addr);
+    if (!uri.ok()) return "";
```

Comment:
I think it's probably fine to just fail here.  Let's just remove this line.

Reply:
Done.